src/Components/Aspire.Keycloak/Aspire.Keycloak.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think we should consider naming this something more specific so that it's clear it's not a client for the Keycloak admin API, but rather a package for easily configuring an ASP.NET Core app to use Keycloak as an IDP. Perhaps `Aspire.Keycloak.Authentication`?

@eerhardt thoughts?

Reply:
Agreed that `Aspire.Keycloak` is a pretty broad/general name. I'm not sure I have a great suggestion. But `Aspire.Keycloak.Authentication` seems better.

Reply:
@julioct - thoughts on the naming here?